### ISSUE TITLE
[General] Option to disable having to use the default leagues

### ISSUE THREAD

**Issue by lorenzodb1** [probberechts/soccerdata#681]
`LEAGUE_DICT` provides a list of default leagues that will be used, along with the ones defined in `league_dict.json`. 

```
LEAGUE_DICT = {**LEAGUE_DICT, **json.load(json_file)}
```

This can be a bit annoying, as I'd have to remove each of the leagues that I don't need individually, thus making my code _dirty_.

Would it be possible to add an option (maybe as an environment variable, but that's just an idea) to disable this behaviour and only use the leagues in `league_dict.json` instead?

**Comment by probberechts** [comment]
That's a good suggestion. I think I would implement this by creating a default `league_dict.json` file if it does not exist yet and move the league config that is now in `_config.py` to that default `league_dict.json` file.

I don't have plan to implement this myself in the short term, but feel free to create a pull request.

**Comment by lorenzodb1** [comment]
How about letting whatever is specified in `league_dict.json`, if present, override the default list of leagues?

**Comment by probberechts** [comment]
I don't think that would provide the best user experience. Most users want to **add** additional leagues, not remove the default ones.

**Comment by lorenzodb1** [comment]
Ok so I'm starting to think about how I could implement this. If we were to with the solution you suggested, we'd have to create the default league file at `make`, as the position of that file isn't fixed. I'm not sure how this can be achieved in a clean way.

What do you think about using an environment variable to let the user decide whether they want to include the default set of leagues or not?
